### PR TITLE
Update the project indexer and persister

### PR DESCRIPTION
**Pull request by jitendrac-egov**

### CODE REVIEW

File: egov-indexer/project-indexer.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@ ServiceMaps:
     - topic: save-project-staff-topic
       configKey: INDEX
       indexes:
-        - name: projectStaffIndex-v1
-          type: project-staff
+        - name: projec-staff-index-v1
```
Comment (egov-joy) [inline egov-indexer/project-indexer.yml:8]: missing `t`
should be `project-staff-index-v1`

File: egov-indexer/project-indexer.yml [hunk 2]
```diff
@@ -15,8 +15,28 @@ ServiceMaps:
     - topic: update-project-staff-topic
       configKey: INDEX
       indexes:
-        - name: projectStaffUpdateIndex-v1
-          type: project-staff-update
+        - name: projec-staff-index-v1
```
Comment (egov-joy) [inline egov-indexer/project-indexer.yml:18]: missing `t`

File: egov-indexer/project-indexer.yml [hunk 3]
```diff
@@ -15,8 +15,28 @@ ServiceMaps:
...
+          type: projectStaff
+          id: $.id
+          isBulk: true
+          jsonPath: $.*
+          timeStampField: $.auditDetails.createdTime
+
+    - topic: save-project-beneficiary-topic
+      configKey: INDEX
+      indexes:
+        - name: projec-beneficiary-index-v1
```
Comment (egov-joy) [inline egov-indexer/project-indexer.yml:28]: missing `t`

File: egov-indexer/project-indexer.yml (hunk 1 above)
Comment (jitendrac-egov) [reply egov-indexer/project-indexer.yml]: Fixed typo

File: egov-indexer/project-indexer.yml (hunk 2 above)
Comment (jitendrac-egov) [reply egov-indexer/project-indexer.yml]: Fixed typo

File: egov-indexer/project-indexer.yml (hunk 3 above)
Comment (jitendrac-egov) [reply egov-indexer/project-indexer.yml]: Fixed typo

**Review by jayantp-egov — APPROVED** [review]

**Pull request merged**